test/views/declaration/AdditionalInformationRequiredViewSpec.scala

```diff
@@ -44,7 +44,7 @@ class AdditionalInformationRequiredViewSpec extends UnitViewSpec with ExportsTes
     "have correct message keys" in {
 
       messages must haveTranslationFor("declaration.additionalInformationRequired.title")
-      messages must haveTranslationFor("supplementary.consignmentReferences.heading")
+      messages must haveTranslationFor("supplementary.items")
```

Comment:
Why you changed this? I can't see which place you changed to use `supplementary.items`

Reply:
The page uses "supplementary.items" for the section heading.  The test was asserting the wrong string.